index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+<link rel="stylesheet" href="C:\Users\Genaro\Desktop\css\bootstrap.css">

```

Comment:
You are referencing a file that doesn't exists. There's no such file called `bootstrap.css` nor `bootstrap-grid.css`. Also you are referencing an absolute path to your computer, so you need to use relative paths. https://www.coffeecup.com/help/articles/absolute-vs-relative-pathslinks/